viewer/daily_viewer.py

```diff
@@ -63,14 +59,30 @@ def set_globals():
             client_version=_DB_VERSION,
         )
 
+    # Find all available revisions
+    if not len(REVISIONS):
```

Comment:
```suggestion
    if not REVISIONS:
```
I think the most pythonic way to check if there's anything in the list is just to see  fit evaluates to True or False (empty or not empty), rather than having to call `len()`